README.md

```diff
@@ -70,7 +68,7 @@ apk add ko
 With Go 1.18+, build and install the latest released version:
 
 ```
-go install github.com/google/ko@latest
+go install github.com/ko-build/ko@latest
```

Comment:
This one will actually have to wait until we rename the Go import path. Even though the repo name has changed, as long as the GH redirect is in place, Go considers this `github.com/google/ko`.

Reply:
I couldn't find any discussion, is there one about doing this anywhere (renaming the go import paths)? Or should someone start one and / or help PR that change?

Reply:
This is roughly sketched out in https://github.com/ko-build/ko/issues/791 

I don't have a set timeline for it yet, I think a couple other things have to happen first. I'd guess some time after Kubecon we'll deprecate `github.com/google/ko` and point folks to importing `ko.build/pkg/build` and `go install ko.build@latest`.

As part of that we'll also move the ko.build website into this repo (#749) which also sets the vanity importpath meta tags.